src/runtime_src/core/pcie/linux/shim.cpp

```diff
@@ -634,7 +669,7 @@ int shim::xclGetDeviceInfo2(xclDeviceInfo2 *info)
     info->mHALMinorVersion = XCLHAL_MINOR_VER;
     info->mMinTransferSize = DDR_BUFFER_ALIGNMENT;
     info->mDMAThreads = 2;
-    xclSysfsGetDeviceInfo(info);
+    // xclSysfsGetDeviceInfo(info);
```

Comment:
@AShivangi , why do you comment out this line? this seems to cause the bandwith test to core dump.